tests/ecmascript/test-dev-syntax-error-line.js

```diff
@@ -21,8 +21,7 @@ function test(inp) {
         print('never here');
     } catch (e) {
         // Match against current syntax
-        res = /^.*\(line (\d+)\)$/.exec(e.message);
-        print('line', res ? res[1] : 'n/a');
+        print('line', e.lineNumber);
```

Comment:
Here the idea of the testcase is to cover the line number in the `.message` property, not the `.lineNumber` property, so the best fix would be to fix the pattern.

Reply:
Fixed